web/src/components/cartSummary.js

```diff
@@ -9,13 +18,56 @@ class CartSummary extends Component {
         };
     }
 
+    receipt = function() {
```

Comment:
Since we're using an arrow function, change the form of all functions to arrow function.